components/Typeahead/_typeahead.scss

```diff
@@ -111,6 +114,7 @@
 
   .bx--typeahead-items--expanded {
     height: auto;
+    z-index: 100;
```

Comment:
Could you switch this to `z-index: z('dropdown');` and add  `@import 'carbon-components/scss/globals/scss/layout'`? Just trying to keep consistency between libraries